src/test/Testcafe/page-objects/loginPage.js

```diff
@@ -0,0 +1,19 @@
+import { Selector, t } from 'testcafe';
+
+
+
+export default class loginPage {
```

Comment:
`LoginPage`
Classnames should start with a capitalized letter.

Reply:
Fixed in new commit